components/salesforce_rest_api/actions/create-user/create-user.mjs

```diff
@@ -0,0 +1,91 @@
+import common from "../common/base.mjs";
+import user from "../../common/sobjects/user.mjs";
+import {
+  pickBy, pick,
+} from "lodash-es";
+import { toSingleLineString } from "../../common/utils.mjs";
+
+const { salesforce } = common.props;
+
+export default {
+  ...common,
+  key: "salesforce_rest_api-create-user",
+  name: "Create User",
+  description: toSingleLineString(`
+    Creates a Salesforce user.
+    See [User SObject](https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_objects_user.htm)
+    and [Create Record](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/dome_sobject_create.htm)
+  `),
+  version: "0.0.1",
+  type: "action",
+  props: {
+    salesforce,
+    Alias: {
```

Comment:
All prop names must start with lowercase

Reply:
This suggestion doesn't make sense, is inconsistent with existing practices within the larger component, and Salesforce in general.

Reply:
Hi @jhaisley, this suggestion makes sense because we follow the [Pipedream Component API Reference](https://pipedream.com/docs/components/api/#component-structure) not the Salesforce one.
LMK if you have any doubts

Reply:
All of the actions and object definitions in [salesforce_rest_api](https://github.com/PipedreamHQ/pipedream/tree/master/components/salesforce_rest_api/) match the external system (Salesforce).  Matching the case-sensitive capitalization of the external system can enhance clarity, especially when this would be the only action within the component to not follow the Salesforce standard.  Worse, although in general Salesforce isn't case sensitive with regards to their API, there are specific functions that are case sensitive, such as when comparing something with their declarative tools like Flow. 

@malexanderlim & @jcortes have both modified other parts of this component within the last few months without following the lowercase standard

I have a goal to add a lot of missing functionality to the Salesforce component in the near future, and I'd like to be able stick with the existing style of the component.

Reply:
I don't know if adapting the code to each external system we integrate is a best practice. But if this component has already been created this way, fine, let's move on.